samples/client/petstore/java/okhttp-gson/src/test/java/org/openapitools/client/model/PropertyNameCollisionTest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * OpenAPI Petstore
+ * This spec is mainly for testing Petstore server and contains fake endpoints, models. Please do not use this for any other purpose. Special characters: \" \\
+ *
+ * The version of the OpenAPI document: 1.0.0
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+
+package org.openapitools.client.model;
+
+import com.google.gson.TypeAdapter;
+import com.google.gson.annotations.JsonAdapter;
+import com.google.gson.annotations.SerializedName;
+import com.google.gson.stream.JsonReader;
+import com.google.gson.stream.JsonWriter;
+import java.io.IOException;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Model tests for PropertyNameCollision
+ */
+public class PropertyNameCollisionTest {
+    private final PropertyNameCollision model = new PropertyNameCollision();
+
+    /**
+     * Model tests for PropertyNameCollision
+     */
+    @Test
+    public void testPropertyNameCollision() {
+        // TODO: test PropertyNameCollision
```

Comment:
Intended to be done before submission?

Reply:
I tested the new option in the java (okhttp-gson) petstore sample instead with a new schema. Will remove this file in my next PR. 

Thanks for reviewing.

Reply:
actually this is the auto-generated test file so we will keep it. the test is added to https://github.com/OpenAPITools/openapi-generator/pull/16103/files/48c9550264c686d154be329a0984142026537e5f#diff-54b45af594b56ff301edf017e96c2786680d87f713d287fcbf5a1429072674b3R8.

the output won't compile without using the nameMapping option in Java okhttp petstore samples.